infrasec/aws/vpcs.md

```diff
@@ -0,0 +1,81 @@
+# [AWS](./README.md) / VPC Configuration
+
+## VPC basics
+
+A VPC is a "Virtual Private Cloud".
+This is a virtual network defined in AWS.
+Each VPC is defined in a specified [region](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions).
+
+The VPC needs a defined CIDR block.
+These are private network blocks and need to have enough ip addresses for whatever you plan to deploy.
+When in doubt, use a `/16`.
+
+Note that each region has multiple [Availability Zones](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-availability-zones) and the ip space will be subdivided into subnets across these availabilty zones.
+
+The following are objects associated with VPCs that can be configured.
+
+### Subnets
+
+A subnet is a logical subdivision of an IP network.
+Generally, you will create a pair of public and private subnets in each availability zone for the region.
```

Comment:
I suggest we also give some guidance on using either `/22` or `/24` for subnets as a default.